clang/test/CodeGen/overflow-idiom-exclusion-fp.c

```diff
@@ -0,0 +1,83 @@
+// Check for potential false positives from patterns that _almost_ match classic overflow-dependent or overflow-prone code patterns
```

Comment:
In general, I think it's expected the RUN lines will be first and any comments about the test can come afterwards. It allows the comments to be set out more prominently. (Here and the next testcase.)